tests/jit/jit_models_test.py

```diff
@@ -132,7 +132,7 @@ def test_enhancement_model(small_model_params, model_def, test_data):
 def test_dcunet_model(test_shape: Tuple, matching_samples):
     n_samples = 5010
     device = get_default_device()
-    model = DCUNet(architecture="mini").eval().to(device)
+    model = DCUNet(architecture="mini", masknet_kwargs={"fix_length_mode": "pad"}).eval().to(device)
```

Comment:
This is very ugly, isn't it?..

Reply:
I agree, I think I did it this way because there where some argument overriding issues with the DCCRN constructor but I don't remember the details and I'm sure we can also get it to work without the extra level. Would you suggest to add the masknet kwargs to the model kwargs, ie. flat? Seems to least ugly to me

Reply:
Yes, this will be much better IMO.